workload/terraform/greenfield/readme.md

```diff
@@ -44,8 +44,8 @@ This folder is laid out hierarchically so that different levels of modules may b
 | ------------------- | ------------------------------------------------------------ |
 | [modules](../modules)            | This folder contains re-usable modules that create infrastructure components that are used to compose more complex scenarios |
 | [ADDS scenarios](./ADDSscenario/readme.md)  | This folder contains scenario root modules that deploy AVD with ADDS join session host. |
-| [AAD scenarios](./AADscenario/readme.md)  | This folder contains scenario root modules that deploy AVD with Microsoft Entra ID join session host. |
-| [AAD Zero Trust scenarios](./zerotrust/readme.md)  | This folder contains scenario root modules that deploy AVD with Microsoft Entra ID join session host following zero trust principles. |
+| [EntraID scenarios](./AADscenario/readme.md)  | This folder contains scenario root modules that deploy AVD with Microsoft Entra ID join session host. |
```

Comment:
Maybe in a followup PR you can rename this folder, might be easier to track as a second PR instead of hoping git detects the `mv` properly.

```suggestion
| [Microsoft Entra ID scenarios](./AADscenario/readme.md)  | This folder contains scenario root modules that deploy AVD with Microsoft Entra ID join session host. |
```